widgetastic_patternfly.py

```diff
@@ -1123,3 +1124,110 @@ def item_select(self, item, handle_alert=None):
             except UnexpectedAlertPresentException:
                 self.logger.warning('There is an unexpected alert present.')
                 pass
+
+
+class Dropup(Dropdown):
+    """Represents the Patternfly/Bootstrap dropup.
+
+    Args:
+        text: Text of the button, can be the inner text or the title attribute.
+    """
+    BUTTON_DIV_LOCATOR = (
+        './/div[contains(@class, "dropup") and ./button[normalize-space(.)={0} or '
+        'normalize-space(@title)={0}]]')
+
+    def __init__(self, parent, text, logger=None):
+        super(Dropup, self).__init__(parent, text=text, logger=logger)
+        self.text = text
+
+
+class CheckBox(BaseInput):
```

Comment:
It's already implemented here https://github.com/RedHatQE/widgetastic.core/blob/master/src/widgetastic/widget.py#L592